ci/cvmfs/snapshotter_container.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
```

Comment:
The file needs to be executable, i.e. run `chmod +x snapshotter_container.sh` and re-commit